watertap/core/util/tests/test_initialization.py

```diff
@@ -11,9 +11,9 @@
 #################################################################################
 
 import pytest
-
+from pyomo.common.errors import InfeasibleConstraintException
 from pyomo.environ import ConcreteModel, Var, Constraint, ConstraintList
-
+import re
 from watertap.core.solvers import get_solver
```

Comment:
I think when we got our own solver, someone did a ctrl+F, "replace all" resulting in this out-of-order import for the solver (i.e., grouped with `idaes` imports rather than `watertap` imports).

Could be corrected in this file or not for this PR, but should be a global change we make at some point (does pylint not check this sort of thing???)

Reply:
I think that this check may be in place on the IDAES repo but for some reason does not come into play here. Never delved into the pylint workflow so unsure whether there is something different between pylint configurations on IDAES and this repo.